src/Setup/WixBA/WixBA.cs

```diff
@@ -1,5 +1,7 @@
 // Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
 
+// WixBAFacelift TODO list:
+// Compile against NetFx452 (supported).  Moved to Net45 to check out using WindowChrome, then backed out Chrome.  check with wix-devs, then update Bundle when framework consenuse exist.
```

Comment:
Do we have to? My hope is that if we find a NETFX v4.0 that we'll be happy to use that. Do we need v4.5+?